Game/MyProject/Game/Helper/MyHelperSceneNode.h

```diff
@@ -0,0 +1,30 @@
+//-------------------------------------------------------------------------------
+//-- MyHelperSceneNode.h --------------------------------------------------------
+//-------------------------------------------------------------------------------
+#pragma once
+//-------------------------------------------------------------------------------
+//-- Dependencies ---------------------------------------------------------------
+//-------------------------------------------------------------------------------
+#include "SceneNode.hpp"
+#include "CameraSceneNode.hpp"
+//-------------------------------------------------------------------------------
+//-- Typedefs, Structs ----------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+//-------------------------------------------------------------------------------
+//-- Classes --------------------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+class MyHelperSceneNode:public SceneNode
+{
+public:
+	
+    MyHelperSceneNode(const std::string& name,RenderPacket renderPacket);
+    ~MyHelperSceneNode(){}
```

Comment:
When doing destructors in an inheritance chain, you either mark it virtual, or final